docker/Dockerfile

```diff
@@ -38,10 +40,35 @@ RUN if [ -z "$DISABLE_MINIFY" ]; then cd meshcentral/translate && node translate
 # translate
 RUN if [ -z "$DISABLE_TRANSLATE" ]; then cd meshcentral/translate && node translate.js translateall; fi
 
+# cleanup
+RUN rm -rf /opt/meshcentral/meshcentral/docker
+RUN rm -rf /opt/meshcentral/meshcentral/node_modules
+
```

Comment:
you could also remove the docs folder as it only contains the md files and powerpoint files

Reply:
the docs/ folder is in the .dockerignore file, it should not be copied at all...